src/mintpy/utils/time_func.py

```diff
@@ -111,6 +114,12 @@ def inps2model(inps, date_list=None, print_msg=True):
             if not (ymin < ptime.yyyymmdd2years(d_step) < ymax):
                 raise ValueError(f'input step date ({d_step}) exceeds date limit: ({dmin} / {dmax})!')
 
+    if inps.polyline:
+        # check 1 - min/max limit
+        for d_start in inps.polyline:
+            if not (ymin < ptime.yyyymmdd2years(d_start) < ymax):
+                raise ValueError(f'input step date ({d_start}) exceeds date limit: ({dmin} / {dmax})!')
```

Comment:
input "polyline" date here?

Reply:
I only spot this printed message typo. Otherwise, you can go ahead to merge it!

Reply:
Fixed. Thanks, kai.